cursos/kubernetes/03_configuracion/12_practica_guiada_assing_pods_to_nodes.md

```diff
@@ -0,0 +1,441 @@
+> Necesita traducción al gallego 👀
```

Comment:
Es necesario tener duplicado el archivo en español para usarlo como página por defecto? En último caso, no podemos crear unos symbolic links al menos? Parece muy difícil de mantener 😿

Reply:
> Es necesario tener duplicado el archivo en español para usarlo como página por defecto? En último caso, no podemos crear unos symbolic links al menos? Parece muy difícil de mantener crying_cat_face

For the translation in docsify I was guided by this experience: https://evilmartians.com/chronicles/easy-multi-language-multi-version-documentation-with-docsify-git-and-github-actions#dealing-with-translations

I understand that almost all the structure could be done by symbolic links, except the index, the sidebar and the custom elements. In any case, every change of translation should be done in both options. 

With other translation experiences in github I have seen that the mechanics is very similar when the translation starts with the repository already on the web. 

The ideal would be to separate all the content of the configuration, so that each language, including Galician, would have its own folder (gl, es, en...)  as is the case of helm documentation: https://github.com/helm/helm-www/tree/main/content

Reply:
Anyway, I follow the line you tell me. Whatever you think is best. Always

Reply:
Sorry @manuelver, I did not understood very well the current structure, default is galician but that file needs to be translated, makes sense... I think there is no need to refactor the dir structure right now. I will try to translate it to galician.

Reply:
We can consider changing the structure to make it simpler. This will be necessary if we want to design this training to be edited by people outside Prefapp. We propose it in the following daily.

Thank you very much for your contributions.  We will continue!  :rocket: